lib/selector/tokenizer/tokenizer.go

```diff
@@ -55,7 +55,7 @@ type Token struct {
 
 const (
 	// LabelKeyMatcher is the base regex for a valid label key.
-	LabelKeyMatcher = `([a-zA-Z0-9_./-]{0,253}/)?[a-zA-Z0-9]([a-zA-Z0-9_.-]{0,61}[a-zA-Z0-9])?`
+	LabelKeyMatcher = `[a-zA-Z0-9_./-]{1,512}`
```

Comment:
We've had to do this a few times now, and I _think_ there is another place that needs to be updated IIRC.